frontend/src/components/DonationButton.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./styles/DonationButton.scss";
+
+export default function DonationButton(props) {
+  return (
+    /* .map THIS COMPONENT TO GENERATE DONATION TO GENERATE DONATION POSSIBILITIES*/
```

Comment:
Do not leave instructions for other devs behind you: 
"do or do not, but there is no should"
          -Yoda (almost)-